### DISCUSSION TITLE
Is carbon-lang written in c or c++?

### DISCUSSION THREAD

**Discussion by Pgirardin** [carbon-language/carbon-lang#1815] [General]
Is carbon-lang written in c or c++?

**Comment by Pgirardin** [comment]
🤔🤔

**Reply by Delphinidae84** [reply]
You can see the languages used in this repository on its main page:

![grafik](https://user-images.githubusercontent.com/38644741/181774219-e6db679d-a592-47ac-928a-0c1c45c8c332.png)

**Comment by prathmesh-b** [comment]
It's written c++ that's why they are replacing it.

**Reply by nacaclanga** [reply]
I guess they white it in C++, rather them any other language like C or Ocaml, so they have one example where the can try out the migration on an actual codebase.
